src/current/cockroachcloud/metrics-essential.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Essential Metrics for CockroachDB Standard Deployments
+summary: Learn about the recommended essential metrics for monitoring your CockroachDB Standard cluster.
+toc: true
+---
+
+These essential CockroachDB metrics let you monitor your CockroachDB {{ site.data.products.standard }} cluster. The metrics are available on graphs on the tabs listed in the **Metrics tabs** column. Where **Custom** is listed, the metric may be graphed in a [**Custom Metrics Chart**]({% link cockroachcloud/custom-metrics-chart-page.md %}). The **Usage** column explains why each metric is important to visualize and how to make both practical and actionable use of the metric in a production deployment.
+
+{% assign types = site.data.metrics | map: "metric_type" | uniq %}
+{% comment %} Fetch the list of all metric types {% endcomment %}
+
+{% for t in types %} {% comment %} Iterate through the types. {% endcomment %}
+
+## {{ t }}
+
+    {% assign metrics = site.data.metrics | where: "metric_type", t | sort: "metric_id" | where_exp: "metrics", "metrics.deploy_standard == true"%}
+    {% comment %} Fetch all metrics for that metric_type. {% endcomment %}
+
+<table>
+    <thead>
+        <tr>
+            <td><b>CockroachDB Metric Name</b></td>
+            <td><b>Short Name</b></td>
+            <td><b>Description</b></td>
+            <td><b>Usage</b></td>
+            <td><b>Metrics tab</b></td>
+        </tr>
+    </thead>
+    <tbody>    
+    {% for m in metrics %} {% comment %} Iterate through the metrics. {% endcomment %}
+        {% assign metrics-list = site.data.metrics-list | where: "metric", m.metric_id %}
+        {% comment %} Get the row from the metrics-list with the given metric_id. {% endcomment %}
+        {% comment %} The metrics-list is generated using: cockroach gen metric-list --format=csv > metrics-list.csv and then changing the case of the headers to lowercase to work with liquid. {% endcomment %}
+        {% assign tab_array = m.metric_ui_tab | remove: '["' | remove: '"]'| split: "," %}
+        
+            <tr>
+            <td><div id="{{ m.metric_id }}" class="anchored"><code>{{ m.metric_id }}</code></div></td>
+            <td>{{ m.short_name }}</td>
+            <td>{{ metrics-list[0].description}}</td>
+            <td>{% include metrics-usage/{{ m.metric_id }}.md %}</td>
+            <td>{% for t in tab_array %}
+                    {% if t contains "Custom" %}
+                        <a href="https://www.cockroachlabs.com/docs/cockroachcloud/custom-metrics-chart-page">{{ t | remove: '"' | strip }}</a>{%- unless forloop.last -%}, {% endunless %}
+                    {% else %}
+                        <a href="https://www.cockroachlabs.com/docs/cockroachcloud/metrics-{{ t | remove: '"' | strip | downcase | replace: ' ', '-' }}">{{ t | remove: '"' | strip }}</a>{%- unless forloop.last -%}, {% endunless %}
```

Comment:
the generated links are 404'ing for me in the render

specifically, if you look at the right-most column in the first row on [this rendered view of the essential metrics page](https://deploy-preview-18362--cockroachdb-docs.netlify.app/docs/cockroachcloud/metrics-essential.html) and click the generated link which has the following text (minus some tracking junk which I stripped off)

https://www.cockroachlabs.com/docs/cockroachcloud/metrics-sql

it 404s

OH I FIGURED IT OUT 🤣 

the links are hardcoded to our website, but these pages do not yet exist on our website

a POSSIBLE action would be to revise this and other generated links to use the `link` syntax - HOWEVER I doubt that would be easy to get working so maybe just ignore

I'm leaving this comment here at least as a reminder of this behavior for future reference during reviews of these generated pages.  Sorry if you told me already and I forgot? and had to rediscover this